Gemfile

```diff
@@ -35,7 +39,7 @@ group(:development, :test) do
   # be removed here *yet* due to TravisCI / AppVeyor which call:
   # bundle install --without development
   # PUP-7433 describes work necessary to restructure this
-  gem "rake", "10.1.1", :require => false
+  gem "rake", :require => false
```

Comment:
I'm thinking this shouldn't be unbounded, but something like `"~> 12.0"`, so we don't break when version 13 comes out